README.md

```diff
@@ -38,11 +38,11 @@
 
 Several quick start options are available:
 
-- [Download the latest release.](https://github.com/twbs/bootstrap/archive/v4.0.0-beta.3.zip)
+- [Download the latest release.](https://github.com/twbs/bootstrap/archive/v4.0.0.zip)
 - Clone the repo: `git clone https://github.com/twbs/bootstrap.git`
-- Install with [npm](https://www.npmjs.com/): `npm install bootstrap@4.0.0-beta.3`
-- Install with [yarn](https://yarnpkg.com/): `yarn add bootstrap@4.0.0-beta.3`
-- Install with [Composer](https://getcomposer.org/): `composer require twbs/bootstrap:4.0.0-beta.3`
+- Install with [npm](https://www.npmjs.com/): `npm install bootstrap@4.0.0`
```

Comment:
`npm install bootstrap` should be enough ?

Reply:
Yeah, we can clean that up afterwards. Let's roll with it like we've done the last couple releases :).

Reply:
Nevermind, see our Slack! Updating this and the download page.